ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -106,7 +106,7 @@ export const AnacondaWizard = ({ onAddErrorNotification, title }) => {
         {
             component: InstallationDestination,
             id: "installation-destination",
-            label: _("Storage configuration"),
+            label: _("Installation destination"),
```

Comment:
I am fixing it in https://github.com/rhinstaller/anaconda/pull/4122. Which issue is it?